holoviews/plotting/mpl/renderer.py

```diff
@@ -318,3 +321,4 @@ def load_nb(cls, inline=True):
         """
```

Comment:
I assumed you meant a class attribute since that makes sense to me. I've now moved that handling onto the  Renderer baseclass, so that all subclasses share the attribute.